extensionsGallery-insider.json

```diff
@@ -3358,6 +3358,63 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "70",
+					"extensionName": "sql-database-projects",
+					"displayName": "SQL Database Projects",
+					"shortDescription": "The SQL Database Projects extension for Azure Data Studio allows users to develop and deploy database schemas.",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.0",
+							"lastUpdated": "06/24/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
```

Comment:
you should use: "assetType": "Microsoft.VisualStudio.Services.VSIXPackage", the download page is for 3rd party extensions. please refer to other extensions that we own.